qiita_db/sql_connection.py

```diff
@@ -1012,6 +1014,22 @@ def execute_fetchindex(self, idx=-1):
         """
         return self.execute()[idx]
 
+    def _funcs_executor(self, cmds, func_str):
+        error_msg = []
+        for cmd in cmds:
+            try:
+                cmd()
```

Comment:
Please read the documentation. There are ways or making this viable always, check the purge_filepath function.